FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/AuthPickerView.swift

```diff
@@ -38,11 +38,11 @@ extension AuthPickerView: View {
             withAnimation {
               switchFlow()
             }
-          }) {
+          }, label: {
```

Comment:
Sorry, I think I might have been a bit vague when asking for this change.

I think I'd prefer the following syntax:

```
            Button(action: { presentingFilters.toggle() }) {
              Text("Unread")
            }
```